active_directory/metadata.csv

```diff
@@ -11,7 +11,7 @@ active_directory.dra.inbound.objects.remaining_in_packet,gauge,,object,second,"T
 active_directory.dra.inbound.properties.applied_persec,gauge,,,second,The number of changes (per second) to object properties that are applied through inbound replication as a result of reconciliation logic.,-1,active_directory,ad dra properties applied
 active_directory.dra.inbound.properties.filtered_persec,gauge,,,second,The number of changes (per second) to object properties received during the replication that are already made.,-1,active_directory,ad dra properties filtered
 active_directory.dra.inbound.properties.total_persec,gauge,,,second,The total number of changes (per second) to object properties received from replication partners.,-1,active_directory,ad dra properties total
-active_directory.dra.inbound.values.dns_persec,gauge,,,second,"The number of values of object properties received (per second) from replication partners in which the values are for object properties that belong to distinguished names. This number includes objects that reference other objects. Values for distinguished names, such as group or distribution list memberships, are more expensive to apply than other kinds of values because a group or distribution list object can include hundreds or thousands of members. In contrast, a simple object might have only one or two attributes. A high number from this counter might explain why inbound changes are slow to be applied to the database.",-1,active_directory,ad dra inbound dns
+active_directory.dra.inbound.values.dns_persec,gauge,,,second,"The number of values of object properties received (per second) from replication partners in which the values are for object properties that belong to distinguished names. This number includes objects that reference other objects. In contrast, a simple object might have only one or two attributes.",-1,active_directory,ad dra inbound dns
```

Comment:
instead of `In contrast, a simple object might have only one or two attributes` can you add the last line `A high number from this counter might explain why inbound changes are slow to be applied to the database`?